### ISSUE TITLE
readme connection example up to date?

### ISSUE THREAD

**Issue by kushal256** [qasaur/gremgo#5]
Hi,

When I try to connect as in the readme example:

 dialer := gremgo.NewDialer("127.0.0.1") // Returns a WebSocket dialer to connect to Gremlin Server
    g, err := gremgo.Dial(dialer) // Returns a gremgo client to interact with
    if err != nil {
        fmt.Println(err)
        return
    }

I get the following compilation error:
cannot use dialer (type gremgo.Ws) as type gremgo.dialer in argument to gremgo.Dial:
	gremgo.Ws does not implement gremgo.dialer (gremgo.connect method has pointer receiver)

Is this connection example uptodate?

Thanks!

**Comment by qasaur** [comment]
Hello,

Thank you for bringing this issue to light. Upon closer inspection I discovered that the problem seems to have spawned from the dialer creation function (NewDialer) and not the README. I have fixed the bug now in the master branch and it should be working properly.

Thanks,
Marcus

**Comment by kushal256** [comment]
Thanks, looks better!

To get this working, I also had to modify something else:

in client.go:
func NewDialer(szhost string) (dialer Ws) {
	return Ws{host: szhost}  //was hardcoding to 127.0.0.1
}

also from the Readme example, I had to change how to call NewDialer:

dialer := gremgo.NewDialer("ws://localhost:8182")

This project looks awesome!  So glad to see a tinkerpop client for golang that doesn't create a new connection for each query.  Looking forward to playing with this library.
